app/views/super_admin/super_admin.py

```diff
@@ -35,6 +36,7 @@ def index_view():
     rejected_sessions = get_count(DataGetter.get_all_rejected_sessions())
     draft_sessions = get_count(DataGetter.get_all_draft_sessions())
     email_times = DataGetter.get_email_by_times()
+    total_users=registered_users + unverified_users
```

Comment:
@niranjan94  for this I need to make a function for doing total right ?  and is this a wrong way to do ?